Vagrantfile

```diff
@@ -0,0 +1,157 @@
+      echo ' '
+      echo "${PURPLE}UPDATING .BASHRC FILE${NC}"
+      echo "cd /home/ubuntu/#{project_name}" >> /home/ubuntu/.bashrc
+      cd ~/#{project_name}
+      echo "${GREEN}COMPLETED UPDATING .BASHRC FILE${NC}"
+      echo ' '
+      echo "${PURPLE}INSTALLING JAVA${NC}"
+      # Updates apt-get to point to updated java8 package
+      yes | sudo add-apt-repository ppa:webupd8team/java
+      # Updates apt-get
+      sudo apt-get update
+      # Set debconf config file to mark java8 license prompts as completed
+      # https://askubuntu.com/questions/190582/installing-java-automatically-with-silent-option#
+      # Install java8 without prompts
+      echo debconf shared/accepted-oracle-license-v1-1 select true | sudo debconf-set-selections
+      echo debconf shared/accepted-oracle-license-v1-1 seen true | sudo debconf-set-selections
+      yes | sudo apt-get install oracle-java8-installer
+      echo "${GREEN}COMPLETED INSTALLING JAVA${NC}"
+      echo ' '
+      echo "${PURPLE}INSTALLING RVM & RUBY${NC}"
+      yes | sudo apt-get install gnupg2
+      gpg2 --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+      # Current issue with rvm: https://github.com/rvm/rvm/issues/4068
+      # use this patch until fixed
+      curl -sSL https://raw.githubusercontent.com/wayneeseguin/rvm/stable/binscripts/rvm-installer | bash -s stable --ruby=2.4
+      # uncomment and use below after fixed
+      # curl -sSL https://get.rvm.io | bash -s stable --ruby=2.4
+      source /home/ubuntu/.rvm/scripts/rvm
+      echo "${GREEN}COMPLETED INSTALLING RVM & RUBY${NC}"
+      echo ' '
+      echo "${PURPLE}INSTALLING GEMS${NC}"
+      gem install bundler --no-rdoc --no-ri
+      # also install libcurl3, libcurl3-gnutls, & libcurl4-openssl-dev
+      # which are needed by the "curb" gem, used in some 3rd party rake tasks--tho not actually sure about neo4j
+      yes | sudo apt-get install libcurl3 libcurl4-openssl-dev
+      bundle install
+      gem install overcommit --no-rdoc --no-ri
+      overcommit --install
+      echo "${GREEN}COMPLETED INSTALLING GEMS${NC}"
+  SHELL
+
+  unless Dir["./db/neo4j/development"].any?
+    # With default configuration, Neo4j only accepts local connections (which, in this case, means local to the VM).
+    # Vagrant has been set up to forward connections local to the VM to the appropriate ports on the host machine.
+    # As such, we need to change neo4j's conf to accept non-local connections if we want to be able to connect to
+    # neo4j from a browser on the host machine:
+    update_neo4j_conf = <<-TEXT
+      require 'fileutils'
+      require 'tempfile'
+
+      t_file = Tempfile.new('neo4j.conf')
+      File.open('./db/neo4j/development/conf/neo4j.conf', 'r+').each do |line|
+        case line.chomp
+        when '#dbms.connectors.default_listen_address=0.0.0.0'
```

Comment:
edit the neo4j.conf file to accept non-local connections (necessary for port forwarding)